### ISSUE TITLE
Edits cards, not just appearance in card editor

### ISSUE THREAD

**Issue by dgbeecher** [simgunz/anki21-addons_editor-max-image-height#1]
Hi,
I previously used this add-on for Anki 2.0 and found it quite useful. 
I just today installed it for Anki 2.1. The behavior seems to significantly have changed: it's added a [style="max-height: 200px;"] tag to the html of all my cards that have images. (Or at least to all cards that I've edited since installing the add-on... I'm still trying to figure out what's happened.) This means images have a max height of 200px even when I'm studying. Whereas I only wanted a max image height while viewing notes in the browser / card editor.
I do have the ImageResizer add-on installed, which is another posible culprint. But it is set to resize images to a max height of 400 px. The Max Image Height add-on is set to a max height of 200 px, which is why I'm inclined to blame it for what's happened.
I'll disable both for now, and see if there's a way I can fix my cards.
I assume this is not the hoped-for behavior of this add-on. Please let me know if I can help figure out what's gone wrong.
Thanks,
Devin

**Comment by simgunz** [comment]
This addon does not modify the card collection so it should not edit the cards in any way.

Can you try to repeat the steps that lead to the problem 
- first by removing ImageResizer and having only editor-max-image-height
- then repeat by removing editor-max-image-height and installing ImageResizer
and see if you can reproduce it?

**Comment by dgbeecher** [comment]
Hello and thank you for your reply. From what I knew about this plugin, I too was sceptical that it would be editing the cards in any way. However, I've followed the steps you suggested, and indeed the problem is reproducible only when editor-max-image-height is installed. It is not reproducible when only ImageResizer is installed. So I guess I'm still suspicious that the problem has something to do with editor-max-image-height.

I'll try to describe in a bit more detail the process that leads to the probem.
1) I'm studying a deck and come to a card with a picture that is more than 200 px high.
2) I open the card editor for that card, either via the "Edit" button on the review screen, or by finding it in the browser.
3) If I immediately return to studying, and view the card again, nothing has changed: the image remains at full height.
4) But if at any point while editing I click into the field with the image, and then return to studying, I notice that the image has shrunk to max 200 px high.
5) If I check on the HTML for that field after clicking into the field, I see this: "style="max-height: 200px;".  
6) If I delete this text and go back to studying, the image is back to its original height.
7) But if at any point I again click into that field while in editing mode, the image's height again gets limited to 200 px.

Note that I don't actually have to edit the card at all for this to happen. I just have to click into (ie, place the cursor within) the field that contains the image. 

Note also that I don't have to be studying cards for this to happen. Ie, steps 1 and 3 are not necessary for the problem arise. These steps just allow me to test when a card has been inadvertently edited.

I've followed this exact workflow after deleting editor-max-image-height, and the above problem does not arise. I've also tried this workflow with editor-max-image-height deleted, but with ImageResizer installed, and the problem does not arise.

This might be helpful too: At one point, *after* deleting editor-max-image-height, but maybe before restarting Anki, I was shown this error message:

```
Debug info:
Anki 2.1.7 (a6c34fd7) Python 3.6.7 Qt 5.12.0 PyQt 5.11.3
Platform: Mac 10.14.2
Flags: frz=True ao=True sv=1

Caught exception:
  File "aqt/webview.py", line 300, in handler
  File "aqt/editor.py", line 327, in oncallback
  File "anki/hooks.py", line 29, in runHook
  File "/Users/dgbeecher/Library/Application Support/Anki2/addons21/229181581/__init__.py", line 23, in setMaxImageHeight
<class 'TypeError'>: 'NoneType' object is not subscriptable
```

I'm not sure exactly what brought the message on, but if I see it again I'll let you know.

Again, please let me know if I can do more to help debug this. Thanks for your help.

**Comment by simgunz** [comment]
Thanks for the info. I will look into this issue soon.

**Comment by dgbeecher** [comment]
Thanks so much!

In case it's useful, here are the other add-ons I have installed. Happy to disable any of these for testing purposes if you think they might be somehow related to the problem.

- Advanced Browser
- Batch Note Editing
- BetterTags
- Card Info During Review
- Multi-column note editor
- Night Mode
- Progress Graphs and Stats for Learned and Matured Cards
- True Retention
- Zoom 21
- avgEase
- review_heatmap

**Comment by simgunz** [comment]
@dgbeecher please let me know if the problem is fixed with the 1.0.1 version of the addon.

**Comment by dgbeecher** [comment]
Hi again. Yes, the problem is fixed! Thank you!
